### DISCUSSION TITLE
配置文件设置

### DISCUSSION THREAD

**Discussion by Yofo-me** [AlistGo/alist#3668] [General]
最新的Alist的config.json没有site_url，新增了assets sub_folder两个键，请问这种情况下如何设置nginx子目录反代alist？我
![2d199c8691ca744b](https://user-images.githubusercontent.com/55388087/222028884-adedce91-b23f-4ab6-868a-272525ca1820.jpg)

![179f5098fbe603da](https://user-images.githubusercontent.com/55388087/222028910-b7a49e26-6580-4d37-af4b-673b5615a4d8.jpg)

![-2a21b5361b55364c](https://user-images.githubusercontent.com/55388087/222028926-728972f2-00f5-45ff-bbf6-b42d6372d203.jpg)

这样设置就启动不了。

如果把config.json里的force的值改成false，那么就会出现500错误，如果只改nginx配置，config.json保持默认，就会出现访问网站结果网页上啥也没有（以前空白)，f12调试发现assets加载不出来。比如说nginx配置的是/alist目录反代alist，访问网站domain.com/alist/时，页面以前空白。assets加载路径应该是domain.com/alist/assets，实际上却是domain.com/assets。

想知道怎么改。

**Comment by Animism001** [comment]
自己在配置文件中添加site_url试试
